app/ide-desktop/lib/dashboard/src/authentication/src/authentication/providers/auth.tsx

```diff
@@ -216,9 +220,26 @@ export const AuthProvider = (props: AuthProviderProps) => {
       navigate(app.LOGIN_PATH);
     });
 
+  const signInWithPassword = async (email: string, password: string) =>
+    cognito.signInWithPassword(email, password).then((result) => {
+      if (result.ok) {
+        toast.success(MESSAGES.signInWithPasswordSuccess);
+        return;
```

Comment:
According to our style guide, we try not to use early returns from functions, as this makes the code harder to read. Use `else` instead if its possible.

Reply:
should be fixed as of latest commit (lint added to detect most early returns)